src/layouts/Static.tsx

```diff
@@ -171,10 +173,14 @@ export const staticComponents = {
   YouTube,
 }
 
-export const StaticLayout = ({
+interface IProps extends PageContent, ChildOnlyProp {
```

Comment:
IMO it'd be better to update the `PageContent` interface and add `lastUpdatedDate` prop as optional if its not required by all layouts, then remove `IProps`

Reply:
I'm game with that, can patch... Similarly I can update this on the other PRs.. I think it would affect the Frontmatter schema PR slightly

Reply:
Oh oops, I actually read this backwards... @nhsz I agree, and have actually already implemented that approach in #42 (Frontmatter schema)... I'll patch this to match here.